content/docs/configuration/issuers.md

```diff
@@ -17,6 +17,7 @@ The following list contains all known cert-manager issuer integrations.
 | 🥈   | command-issuer              | [📄][config:command-issuer]         | [Keyfactor Command][ca:command-issuer]                                 | -                                                 | [✔️][release:command-issuer]         | ✔️              |
 | 🥈   | ejbca-issuer                | [📄][config:ejbca-issuer]           | [EJBCA][ca:ejbca-issuer]                                               | -                                                 | [✔️][release:ejbca-issuer]           | ✔️              |
 | 🥈   | google-cas-issuer           | [📄][config:google-cas-issuer]      | [Google Cloud Certificate<br/>Authority Service][ca:google-cas-issuer] | -                                                 | [✔️][release:google-cas-issuer]      | ✔️              |
+| 🥈   | gs-atlas-issuer             | [📄][config:gs-atlas-issuer]        | [GlobalSign CA][ca:gs-atlas-issuer]                                    | -                                                 | [✔️][release:gs-atlas-issuer]        | ✔️              |
```

Comment:
I changed your issuer from tier-1 to tier-2 because I couldn't find an E2E tutorial for it. 
For tier-1 there should be an up to date tutorial somewhere. Perhaps in the GlobalSign Atlas documentation:
 * https://support.globalsign.com/atlas